File: 41. First Missing Positive/Solution.cpp
class Solution {
public:
    int firstMissingPositive(vector<int>& nums) {
        int m = 1;
        sort(nums.begin(),nums.end());
        for(int i:nums){
            if(i>0){
                if(i==m){m++;}
                else if(m<i){break;}
            }
        }
        return m;
    }
    
};

File: 41. First Missing Positive/Solution.py
class Solution:
    def firstMissingPositive(self, nums: List[int]) -> int:
        m = 1
        nums.sort()
        for i in nums:
            if i>0:
                if i==m:
                    m+=1
                elif m<i:
                    break
        return m
